src/WrapperApp/components/InputEditor/InputFilesEditor.tsx

```diff
@@ -138,18 +145,24 @@ export function InputFilesEditor(props: InputFilesEditorProps) {
 											Delete
 										</Button>
 									)}
+									{isLargeFile && (
+										<Box
+											color='error.main'
+											sx={{
+												ml: 1,
+												display: 'inline-flex',
+												fontSize: 'initial'
+											}}>
+											File is to large and was truncated...
+										</Box>
+									)}
 								</h2>
-
 								<CodeEditor
 									aria-label={name + ' text field'}
-									value={value}
+									value={content}
 									language='sql'
 									placeholder={`Please enter ${name} content.`}
-									onChange={evn =>
```

Comment:
why is this removed ? are we able to submit modified files to the backend ?

Reply:
With this handler present, it will be possible.
It will update the generated value with one present in the editor.

I marked the editor as disabled in the next line, so here it should be handled in a similar way.
It is probably a leftover after refactoring, I previously used two separate templates.
I will fix it